src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private enum Color { Red, Orange, Yellow, Green, Blue }
```

Comment:
`enum Color` must be created as a separate file) Also, all of his constants must be in the upper case
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names